db/seeds.rb

```diff
@@ -5,54 +5,73 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+unless Rails.env.production?
```

Comment:
seeds shouldn't be run in prod so I guess this check is unnecessary?

Reply:
I put it just in case, but you are right. One less condition